src/Microsoft.AspNet.Mvc.ViewFeatures/Resources.resx

```diff
@@ -137,10 +137,7 @@
     <value>An invoker could not be created for the view component '{0}'.</value>
   </data>
   <data name="ViewComponent_CannotFindMethod" xml:space="preserve">
-    <value>Could not find an '{0}' method matching the parameters.</value>
-  </data>
-  <data name="ViewComponent_CannotFindMethod_WithFallback" xml:space="preserve">
-    <value>Could not find an '{0}' or '{1}' method matching the parameters.</value>
+    <value>Could not find an '{0}' or '{1}' method for the view component '{2}'.</value>
```

Comment:
Should stay consistent with elsewhere when saying `view component` vs. `View Component`